application/src/serializer.cpp

```diff
@@ -22,3 +26,31 @@ QString Serializer::getId(const QByteArray &jsonData)
     QString id = jsonObj["id"].toString();
     return id;
 }
+
+QList<Message> Serializer::serializeMessage(const QByteArray &jsonData)
+{
+    QList<Message> messageList;
+
+    QJsonParseError parseError;
+
+    QJsonDocument jsonDoc = QJsonDocument::fromJson(jsonData, &parseError);
+    QJsonObject jsonObj = jsonDoc.object();
+    QJsonValue jsonMessages = jsonObj.value("messages");
+
+    if(jsonMessages.isArray()) {
```

Comment:
```suggestion
    if (jsonMessages.isArray()) {
```
code-style
https://wiki.qt.io/Qt_Coding_Style